Assets/_Application/Layers/LayerTypes/ReferencedLayerData.cs

```diff
@@ -31,8 +31,16 @@ public ReferencedLayerData(string name, LayerGameObject reference) : base(name)
         public ReferencedLayerData(string name, string prefabId, List<LayerPropertyData> layerProperties) : base(name, layerProperties)
         {
             this.prefabId = prefabId;
-            var prefab = ProjectData.Current.PrefabLibrary.GetPrefabById(prefabId);
-            Reference = GameObject.Instantiate(prefab);
+            ProjectData.Current.PrefabLibrary
+                .Instantiate(prefabId)
+                .Then(layerGameObject => OnSuccessfullyInstantiated(layerGameObject, layerProperties));
```

Comment:
ik weet niet zeker of dit altijd goed gaat, nu is er dus een periode dat `Reference` null is, en dat kan problemen geven

Reply:
Goeie vraag! Daar moet ik even rustig voor zitten om te evalueren; want ik zou wel de flow hier kunnen blocken - maar dan kan het een halve minuut of meer geblocked zijn. En dan gaan er andere dingen stuk.

In #429 wordt een blocking loading screen geintroduceerd wat de gebruikersinteracties zal beperken, maar niet die van achtergrond processen 🤔